lib/secure-boot-lib.robot

```diff
@@ -44,6 +44,11 @@ ${EXPIRED_SHA256}=          831c4f0ec9bf4e40a80e15deb16d76a60524e91bd9e9ebe07847
 
 
 *** Keywords ***
+Make Certificate Images For Tests
+    [Documentation]    This keyword generates all the necessary images
+    ...    for secure boot tests.
+    Run    sudo ./scripts/secure-boot/generate-images/wrapper.sh
```

Comment:
You should probably check that return code is 0 (finished correctly)

```
    ${rc}=    Run And Return RC    ./scripts/secure-boot/generate-images/wrapper.sh
    Should Be Equal As Integers   ${rc}   0
```

You would also need to modify scripts so they exit if command fails. You can do that by adding `set -e` at the beginning of each script (wrapper and generate-image). It will make whole script fail immediately if any command fails.
This can be disabled using `set +e` before commands you want to allow to fail and then using `set -e` again after them.

Reply:
adressed in [this commit](https://github.com/Dasharo/open-source-firmware-validation/pull/192/commits/2c746fe7fb34c331de12a48ef66a0aa9eaa5cd63)

Reply:
Works but maybe change to: `${rc}    ${output}=    Run And Return RC And Output`.
This way when there is error you could see why in logs. Currently there is no information.

Reply:
this is a great idea - added [here](https://github.com/Dasharo/open-source-firmware-validation/pull/192/commits/531a320643f8b7fb2630b6fdaa0630fe1f044f66)